scripts/install_build_dependencies.sh

```diff
@@ -16,21 +16,21 @@ apt-get update
 
 # Install G++/GCC cross compilers
 if [ "$DEBIAN_ARCH" == "arm64" ]; then
-  apt-get install -y \
+  apt-get -t testing install -y \
```

Comment:
As we have the configuration of the sources in the scripts/apt/default.pref, isn't it better to move this param there as a set of testing sources? it will look more consistent and allow to use a regular script and individual commands.

Reply:
Removed, as testing has higher priority.